app/experimenter/experiments/serializers.py

```diff
@@ -413,7 +432,7 @@ class Meta:
         )
 
     def get_action_name(self, obj):
-        if obj.is_multi_pref:
+        if obj.is_multi_pref_experiment or obj.is_multi_pref_format:
```

Comment:
Can't we just do the `is_multi_pref_format` check inside of `is_multi_pref_experiment`?

Reply:
kk will do